src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```diff
@@ -11,10 +11,13 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <AnalysisLevel Condition="'$(AnalysisLevel)' == ''">5.0</AnalysisLevel>
```

Comment:
Why not use a single property name `WarnVersion` for both compiler and .NET SDK analyzers? Do you feel people might want to provide different values for either of them? Will two property names confuse users?

Reply:
my thinking was that `AnalysisLevel` controls both which analyzers are active and which warning level is passed to the compiler. `WarnVersion` is only about what the compiler produces. Today both of these properties will have the same value, but in the future `WarnVersion` will be tied to the compiler and `AnalysisLevel` will be tied to the platform

Reply:
Should the difference be commented in the code then?

Reply:
Is `AnalysisLevel` intended to increment with the TFM by default? If so, would it make sense to move the TFM version check into a condition on `AnalysisLevel` (so that it's not defined for old TFMs by default)?

This would make it a little easier for other tools (ILLink in my case) to respect `AnalysisLevel`, by simplifying the mapping to our the tool-specific warning level property the same way that it simplifies the `EnableNETAnalyzers` condition.

Reply:
I'll update the PR